reledpar.dtx

```diff
@@ -1457,14 +1457,14 @@
 % \begin{macro}{\led@error@edtext@later@now}
 %    \begin{macrocode}
 \newcommand{\led@error@edtext@later@now}{%
-  \reledpar@error{You call \the\edtext@later\space \string\edtextlater\space commands but  \the\edtext@now\space \string\edtextnow\space commands}{\@ehc}%
+  \reledpar@error{You called the \the\edtext@later\space \string\edtextlater\space commands, not the \the\edtext@now\space \string\edtextnow\space commands}{\@ehc}% (did I follow your intent correctly here?)
```

Comment:
now,1) all the message of error are in present, not preterit 2) this specific message compare two number, so the "but" is better than the "not the".